src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/LegacyPackageReferenceProject.cs

```diff
@@ -170,7 +174,7 @@ private CentralPackageVersion ToCentralPackageVersion(string packageId, string v
         public override async Task<IEnumerable<PackageReference>> GetInstalledPackagesAsync(CancellationToken token)
         {
             // Settings are not needed for this purpose, this only finds the installed packages
-            return GetPackageReferences(await GetPackageSpecAsync(NullSettings.Instance));
+            return await GetPackageReferences(await GetPackageSpecAsync(NullSettings.Instance));
```

Comment:
Can you achieve the same result with one `await` keyword? 
```
            return await GetPackageReferences(GetPackageSpecAsync(NullSettings.Instance));
```

Reply:
Since GetPackageSpecAync is a async method, I need the `await` no?